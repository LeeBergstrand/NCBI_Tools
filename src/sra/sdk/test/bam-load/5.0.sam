@CO	Unaligned, no pair
@CO	-----------------------------------------------------
@HD	VN:1.0	GO:none	SO:coordinate
@SQ	SN:20	LN:63025520	AS:NCBI37	
SRR006423.6228848	146	*	60079	60	16S35M	=	60025	-89	GATCCTGTCAAAAAACAGGTGGTAAGGAAGGAGAGAGTGAAGGCACTGCCA	%."%/#:#9#""9%7$3<A%%A<#:@$@>AA>=57:<5=$<5)%4<?-:=;	XC:i:35	XT:A:U	NM:i:1	SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:1	XO:i:0	XG:i:0	MD:Z:27A7	RG:Z:SRR006423	OQ:Z:%&"%&#)#,#""+%($&(-%%/,#*-$2-22.-**-2,3&2+&%/II+III
