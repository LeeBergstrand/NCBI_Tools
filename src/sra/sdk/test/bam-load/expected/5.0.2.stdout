     ALIGNMENT_COUNT: 0
          BASE_COUNT: 51
      BIO_BASE_COUNT: 51
      CMP_BASE_COUNT: 51
        COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
              CSREAD: 010312113020211222222020202031011021100000121120232
              CS_KEY: T
           CS_NATIVE: false
      FIXED_SPOT_LEN: 51
         MAX_SPOT_ID: 1
         MIN_SPOT_ID: 1
                NAME: 1
            PLATFORM: SRA_PLATFORM_UNDEFINED
PRIMARY_ALIGNMENT_ID: 0
             QUALITY: 30, 30, 30, 10, 30, 30, 10, 1, 1, 10, 10, 1, 10, 10, 10, 10, 1, 1, 10, 10, 10, 10, 10, 10, 1, 10, 1, 1, 10, 10, 1, 1, 10, 1, 1, 1, 1, 1, 10, 1, 1, 1, 10, 1, 1, 1, 1, 1, 1, 1, 1
           RD_FILTER: SRA_READ_FILTER_PASS
                READ: TGGCAGTGCCTTCACTCTCTCCTTCCTTACCACCTGTTTTTTGACAGGATC
         READ_FILTER: SRA_READ_FILTER_PASS
            READ_LEN: 51
            READ_SEG: [0, 51]
          READ_START: 0
           READ_TYPE: SRA_READ_TYPE_BIOLOGICAL|SRA_READ_TYPE_REVERSE
          SIGNAL_LEN: 0
          SPOT_COUNT: 1
          SPOT_GROUP: SRR006423
             SPOT_ID: 1
            SPOT_LEN: 51
          TMP_KEY_ID: 0
            TRIM_LEN: 51
          TRIM_START: 0

